integration-tests/sandbox-ballot-operation.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { CONFIGS, sleep } from './config';
+
+CONFIGS().forEach(async ({ lib, protocol, setup, rpc }) => {
+  const Tezos = lib;
+  const flextesanet = (rpc === 'http://0.0.0.0:20000') ? it : it.skip;
+
+  Tezos.setRpcProvider(rpc);
+
+  describe(`Ballot operation test (${protocol})`, () => {
+    beforeAll(async (done) => {
+      await setup();
+
+      try {
+        const period = await Tezos.rpc.getCurrentPeriod();
+        await sleep((period.remaining + 5)  * 1000);
+
+        const proposalsOp = await Tezos.contract.proposals({
+          proposals: ['ProtoALphaALphaALphaALphaALphaALphaALphaALphaDdp3zK']
+        });
+
+        await proposalsOp.confirmation();
+      } catch(e) {
```

Comment:
For both tests, after the confirmation, can you verify that `op.status `is `applied `and `hash` is defined?

Reply:
for this operation there is no `status` in `operationResults`